src/tx.rs

```diff
@@ -32,22 +31,22 @@ pub struct UnprovenTransactionInput {
     value: u64,
     blinder: JubJubScalar,
     pk_r_prime: JubJubExtended,
-    sig: SchnorrSig,
+    sig: SignatureDouble,
 }
 
 impl UnprovenTransactionInput {
     fn new<Rng: RngCore + CryptoRng>(
         rng: &mut Rng,
-        ssk: &SecretSpendKey,
+        sk: &SecretKey,
         note: Note,
         value: u64,
         blinder: JubJubScalar,
         opening: PoseidonOpening<(), POSEIDON_TREE_DEPTH, 4>,
         tx_hash: BlsScalar,
     ) -> Self {
-        let nullifier = note.gen_nullifier(ssk);
-        let sk_r = ssk.sk_r(note.stealth_address());
-        let sig = SchnorrSig::new(&sk_r, rng, tx_hash);
+        let nullifier = note.gen_nullifier(sk);
+        let sk_r = sk.sk_r(note.stealth_address());
```

Comment:
Same than in Rusk, we could use nsk or note_sk here.

Reply:
I was planning to change those when the function names also change. With the update to phoenix-core 0.27 or higher